doc/releases/changelog-dev.md

```diff
@@ -19,9 +19,29 @@
   
   [(#1063)](https://github.com/PennyLaneAI/catalyst/pull/1063)
 
+* Fixes an issue where certain JAX linear algebra functions from `jax.scipy.linalg` gave
+  incorrect results when invoked from within a qjit block, and adds support for various
+  other `jax.scipy.linalg` functions, including, but not limited to:
```

Comment:
should we list all the newly supported functions here in changelog? or maybe not list them at all and just say `jax.scipy.linalg` is now supported? @josh146

Reply:
I've listed these specific functions here since these are the ones I have unit tests for. I'm confident all of the `jax.scipy.linalg` will be supported given that we now supply all of the LAPACK symbols that JAX expects, and since the new unit tests touch all of these LAPACK routines, but I haven't tested all of the other JAX linalg functions so I can't claim with certainty that they'll work.

Reply:
For reference, the full list of JAX linalg functions is here: https://jax.readthedocs.io/en/latest/jax.scipy.html#module-jax.scipy.linalg.

Reply:
I see, I'm good with this explicit list then. I will wait for Josh to rule; if he greenlights feel free to resolve this

Reply:
Any particular reason why we choose these particular ones for unit test? (just curious)

Reply:
These were the functions listed on [Shortcut](https://app.shortcut.com/xanaduai/epic/67495/) by @josh146 with the highest-priority to support, with a few others thrown in for good measure. I'm happy to add more tests for other functions if needed!

Reply:
This looks great! thanks @joeycarter

Reply:
It might also be worth updating this quickly: https://github.com/PennyLaneAI/catalyst/blob/joeycarter/jax-lapack-wrappers/doc/dev/jax_integration.rst?plain=1#L91

Reply:
Thanks everyone! :tada:

> It might also be worth updating this quickly: https://github.com/PennyLaneAI/catalyst/blob/joeycarter/jax-lapack-wrappers/doc/dev/jax_integration.rst?plain=1#L91

Fixed, thanks for the reminder!